odxtools/diaglayertype.py

```diff
@@ -3,7 +3,7 @@
 from enum import Enum
 
 
-class DIAG_LAYER_TYPE(Enum):
+class DiagLayerType(Enum):
```

Comment:
Breaking change?

Reply:
Maybe make just a small PR only for rename

Reply:
the problem with this is that in its current form, the patch needs the `ParentRef` class in a separate file (which is done in the first commit of this PR). This which implies quite a bit of additional work for IMO very limited benefits. do you want to have it done that way anyway?